### DISCUSSION TITLE
How to customize the certificate subject

### DISCUSSION THREAD

**Discussion by wuzhong130** [FiloSottile/mkcert#596] [Q&A]
How to customize the certificate subject

**Comment by DoraTheExploder** [comment]
I've discovered that if you use an external tool to generate a Certificate Signing Request (.csr) such as [DigiCert CSR Tool](https://www.digicert.com/easy-csr/openssl.htm) (or do it manually with openssl), you can customize the relevant fields and then pass it as a sort of configuration file to mkcert with `$ mkcert -csr <domain>`.

I have not tested using a CSR for initial Root CA installation, but `$ mkcert --help` implies that it should be possible.
